src/site/stages/build/plugins/create-outreach-assets-data.js

```diff
@@ -49,12 +49,9 @@ function createOutreachAssetsData(buildSettings) {
       entity.derivedFields = { absoluteUrl, fileSize };
     }
 
-    const outreachAssetsFileName = 'generated/outreach-assets.json';
     const serializedOutreachAssets = JSON.stringify(outreachAssets, null, 2);
 
-    files[outreachAssetsFileName] = {
-      contents: Buffer.from(serializedOutreachAssets),
-    };
+    metalsmith.metadata({ outreachAssetsData: serializedOutreachAssets });
```

Comment:
Instead of writing this as a JSON file, we write it to the [body of the page](https://github.com/department-of-veterans-affairs/vets-website/pull/10020/files#diff-ff0ef4a94946b7836966fe63857b76fcR7) so that we don't need to send an HTTP request to get it. 

As an improvement, I think it would be cool if we could convert the JS code into templating to process the `outreachAssets` into the list of images, then just use JS code to paginate it.